bin/generateBatchArmResponses.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 echo "This script generates the ARM pull response files. These can be used for testing where ARM returns a response for an object such as media, transcription document, annotation document or case document to be stored in the ARM storage"
 
-uuid1=$(uuidgen)
+uuid1=$(uuidgen | tr -d '-')
+##uuid1=${uuid1//'-'/''}
```

Comment:
Intentional to leave the commented line here?

Reply:
doh